server/routerlicious/packages/protocol-base/src/blobs.ts

```diff
@@ -42,14 +41,12 @@ export function getGitMode(value: SummaryObject): string {
  * @param value - summary object
  * @returns the type of summary object
  */
-export function getGitType(value: SummaryObject): string {
+export function getGitType(value: SummaryObject): "blob" | "commit" | "tree" | "attachment" {
```

Comment:
In the function definition you have removed the possibility to return "commit", but the return type signature of the function includes "commit".